examples/text_classification/README.md

```diff
@@ -21,8 +21,12 @@ The above command generated the model's state dict as model.pt and the vocab use
  * Create a torch model archive using the torch-model-archiver utility to archive the above files.
  
     ```bash
-    torch-model-archiver --model-name my_text_classifier --version 1.0 --model-file model.py --serialized-file model.pt --source-vocab source_vocab.pt --handler text_classifier --extra-files index_to_name.json
+    torch-model-archiver --model-name my_text_classifier --version 1.0 --model-file model.py --serialized-file model.pt  --handler text_classifier --extra-files "index_to_name.json,source_vocab.pt"
```

Comment:
@dhaniram-kshirsagar Will this require updates to automated tests?

Reply:
It is in place via torchserve_sanity which is part of CI.